### ISSUE TITLE
House price prediction 


### ISSUE THREAD

**Issue by Jenicisprakash** [Jenicisprakash/project#1]
Creating a house price prediction model involves several steps. Here's a simplified example using Python and the scikit-learn library with a linear regression model:

# Import necessary libraries
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.linear_model import LinearRegression
from sklearn.metrics import mean_squared_error
import matplotlib.pyplot as plt

# Sample data (replace with your dataset)
data = {'SquareFeet': [1400, 1600, 1700, 1875, 1100],
        'Bedrooms': [3, 3, 2, 4, 2],
        'Price': [245000, 312000, 279000, 308000, 199000]}

# Create a DataFrame
df = pd.DataFrame(data)

# Split the data into features (X) and target variable (y)
X = df[['SquareFeet', 'Bedrooms']]
y = df['Price']

# Split the data into training and testing sets
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Create a linear regression model
model = LinearRegression()

# Train the model
model.fit(X_train, y_train)

# Make predictions
y_pred = model.predict(X_test)

# Evaluate the model
mse = mean_squared_error(y_test, y_pred)
print(f'Mean Squared Error: {mse}')

# Visualize predictions
plt.scatter(X_test['SquareFeet'], y_test, color='black', label='Actual Prices')
plt.scatter(X_test['SquareFeet'], y_pred, color='blue', label='Predicted Prices')
plt.xlabel('Square Feet')
plt.ylabel('Price')
plt.legend()
plt.show()
```

Replace the sample data with your dataset. This is a basic example, and in a real-world scenario, you would need to handle data preprocessing, feature scaling, and possibly use more advanced algorithms for better accuracy.
